applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -835,10 +894,8 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 static void disconnected_headset_cleanup(uint8_t chan_idx)
 {
 	headsets[chan_idx].headset_conn = NULL;
-	memset(&headsets[chan_idx].sink_stream, 0, sizeof(struct bt_audio_stream));
```

Comment:
Would appreciate an explanation of why these are no longer required.

Reply:
We might no need to clean up the stream structure.
Since that will clean the ops setting (callback hander register), and also clean the qos setting which made by unicast group create API, leads to a recent nightly failed test: audio stream from headset not recover after reset.

Reply:
Yes this should run a reset on the disconnected streams so as to put it back into an initialised state. We only should run the initialise once but can run a reset many times. The stream will be ready for a reconnect then.